drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -294,6 +340,72 @@ local function temp_attr_handler(driver, device, ib, response)
   end
 end
 
+local mired_bounds_handler_factory = function(minOrMax)
+  return function(driver, device, ib, response)
+    if ib.data.value == nil then
+      return
+    end
+    local color_temp_mireds = ib.data.value
+    if (color_temp_mireds < COLOR_TEMPERATURE_MIRED_MIN or color_temp_mireds > COLOR_TEMPERATURE_MIRED_MAX) then
+      device.log.warn_with({hub_logs = true}, string.format("Device reported a min or max color temperature %d mired outside of supported capability range", color_temp_mireds))
+      if (minOrMax == COLOR_TEMP_MIN) then
+        color_temp_mireds = COLOR_TEMPERATURE_MIRED_MIN
+      else
+        color_temp_mireds = COLOR_TEMPERATURE_MIRED_MAX
+      end
+    end
+    local temp_in_kelvin = mired_to_kelvin(color_temp_mireds)
+    set_field_for_endpoint(device, COLOR_TEMP_BOUND_RECEIVED..minOrMax, ib.endpoint_id, temp_in_kelvin, {persist = true})
```

Comment:
Is this change to make it persistent intentional?

Reply:
It was not, and I removed it in the last commit: 8db6cbd25f144d5dac42038ee0ed766cabc54a3d.